cms/envs/test.py

```diff
@@ -1,277 +1,10 @@
-
-# Nose Test Runner
-TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
-
-_SYSTEM = 'cms'
-
-_REPORT_DIR = REPO_ROOT / 'reports' / _SYSTEM
-_REPORT_DIR.makedirs_p()
-_NOSEID_DIR = REPO_ROOT / '.testids' / _SYSTEM
-_NOSEID_DIR.makedirs_p()
-
-NOSE_ARGS = [
-    '--id-file', _NOSEID_DIR / 'noseids',
-    '--xunit-file', _REPORT_DIR / 'nosetests.xml',
-]
-
-TEST_ROOT = path('test_root')
-
-# Want static files in the same dir for running on jenkins.
-STATIC_ROOT = TEST_ROOT / "staticfiles"
-
-GITHUB_REPO_ROOT = TEST_ROOT / "data"
-DATA_DIR = TEST_ROOT / "data"
-COMMON_TEST_DATA_ROOT = COMMON_ROOT / "test" / "data"
-
-# For testing "push to lms"
-FEATURES['ENABLE_EXPORT_GIT'] = True
-GIT_REPO_EXPORT_DIR = TEST_ROOT / "export_course_repos"
-
-# Makes the tests run much faster...
-SOUTH_TESTS_MIGRATE = False  # To disable migrations and use syncdb instead
-
-# TODO (cpennington): We need to figure out how envs/test.py can inject things into common.py so that we don't have to repeat this sort of thing
-STATICFILES_DIRS = [
-    COMMON_ROOT / "static",
-    PROJECT_ROOT / "static",
-]
-STATICFILES_DIRS += [
-    (course_dir, COMMON_TEST_DATA_ROOT / course_dir)
-    for course_dir in os.listdir(COMMON_TEST_DATA_ROOT)
-    if os.path.isdir(COMMON_TEST_DATA_ROOT / course_dir)
-]
-
-# Avoid having to run collectstatic before the unit test suite
-# If we don't add these settings, then Django templates that can't
-# find pipelined assets will raise a ValueError.
-# http://stackoverflow.com/questions/12816941/unit-testing-with-django-pipeline
-STATICFILES_STORAGE = 'pipeline.storage.NonPackagingPipelineStorage'
-STATIC_URL = "/static/"
-
-# Update module store settings per defaults for tests
-update_module_store_settings(
-    MODULESTORE,
-    module_store_options={
-        'default_class': 'xmodule.raw_module.RawDescriptor',
-        'fs_root': TEST_ROOT / "data",
-    },
-    doc_store_settings={
-        'db': 'test_xmodule',
-        'host': MONGO_HOST,
-        'port': MONGO_PORT_NUM,
-        'collection': 'test_modulestore{0}'.format(THIS_UUID),
-    },
-)
-
-CONTENTSTORE = {
-    'ENGINE': 'xmodule.contentstore.mongo.MongoContentStore',
-    'DOC_STORE_CONFIG': {
-        'host': MONGO_HOST,
-        'db': 'test_xcontent',
-        'port': MONGO_PORT_NUM,
-        'collection': 'dont_trip',
-    },
-    # allow for additional options that can be keyed on a name, e.g. 'trashcan'
-    'ADDITIONAL_OPTIONS': {
-        'trashcan': {
-            'bucket': 'trash_fs'
-        }
-    }
-}
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': TEST_ROOT / "db" / "cms.db",
-        'ATOMIC_REQUESTS': True,
-    },
-}
 
 # This hack disables migrations during tests. We want to create tables directly from the models for speed.
 # See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
 MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
```

Comment:
But if the goal of this was to disable migrations, why not just disable them the right way here, and not add the complexity of two different ways to run tests?